Example/Emission/AppDelegate.m

```diff
@@ -63,45 +63,50 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
   emission.switchBoardModule.presentNavigationViewController = ^(UIViewController * _Nonnull fromViewController,
```

Comment:
perhaps this should take an animated option, I could see times when you'd want to handle something non-animated